Finjector.Web/ClientApp/src/components/Entry/EntryEditButtons.tsx

```diff
@@ -44,8 +46,8 @@ const EntryEditButtons = (props: Props) => {
   const copy = () => {
     // create a new chart based on the starting point of current chart
     const chartToSave: Coa = {
-      ...props.savedChart,
-      folderId: props.savedChart.folderId || 0,
+      ...savedChart,
+      folderId: savedChart.folderId || 0,
       id: 0,
       name: `${props.savedChart.name}`,
```

Comment:
not a biggie/i can change later, but the rest of the file still does `props.savedChart`. i've been really liking destructuring at the declaration, like `const EntryEditButtons = ({ chartData, savedChart }: Props) => {` because it shows you the props when you hover using the component elsewhere. i would love to make this a standard we do 👀 
![image](https://github.com/ucdavis/finjector/assets/27025973/f53ab735-e90e-4ecd-9104-ccd50ad4d384)